src/core/messenger/passback.ts

```diff
@@ -205,15 +205,34 @@ const init = (register: RegisterListener): void => {
 										`Passback: ad height is ${adHeight}`,
 									);
 									void fastdom.mutate(() => {
-										const slotHeight = `${
-											adHeight + adLabelHeight
-										}px`;
+										//The 100 is to redue CLS if Teads passes back so we don't resize back to 274px for desktop if we get an mpu
```

Comment:
I know the ticket probably mentions desktop for removing the second shift and don't resize back to `250px` but does that means we keep mobile as it is at the moment? I need some clarification on that.